charms/openfga-k8s/tests/charms/openfga_requires/src/charm.py

```diff
@@ -72,21 +72,24 @@ def _on_openfga_store_created(self, event: OpenFGAStoreCreateEvent):
 
         logger.info("store id {}".format(event.store_id))
         logger.info("token_secret_id {}".format(event.token_secret_id))
+        logger.info("token {}".format(event.token))
```

Comment:
Should we be logging it?

Reply:
yes.. again.. this is purely a test charm.. and logging it here makes it easier for me to follow logs and make sure proper values make it to the other side